src/main/java/tutoraid/logic/parser/CliSyntax.java

```diff
@@ -10,4 +10,5 @@
     public static final Prefix PREFIX_STUDENT_PHONE = new Prefix("sp/");
     public static final Prefix PREFIX_PARENT_NAME = new Prefix("pn/");
     public static final Prefix PREFIX_PARENT_PHONE = new Prefix("pp/");
+    public static final Prefix PREFIX_LIST_ALL = new Prefix("-a");
```

Comment:
I wanted to clarify if this is similar to the '-s' and '-p' flags we use for the other commands? Because the '-s' and '-p' flags are currently handled in TutorAidParser.java.

But if this current implementation works, maybe it is neater to implement '-s' and '-p' like this instead?

Reply:
Yeah I was actually thinking we have SLAP issues with the handling of '-s' and '-p' there. But at the same time, we'd have to combine the add student and add progression into one command possibly? I feel like that might be worse... Maybe we can handle all flags in TutorAidParser but handle the SLAP issues.